example/demo/src/index.ts

```diff
@@ -4,14 +4,15 @@ import { createWSServer, InferModel } from '@rise-tools/server'
 import { models as delivery } from './delivery/ui'
 import { models as inventory } from './inventory/ui'
 import { models as controls } from './ui-controls/ui'
+import { isProcessRunning } from './util'
 
 const models = { ...inventory, ...controls, ...delivery }
 
 const port = Number(process.env.PORT || '3005')
 
 const server = createWSServer(models, port)
 
-if (process.env.NODE_ENV === 'development') {
+if (process.env.NODE_ENV === 'development' && !isProcessRunning()) {
```

Comment:
I think a cleaner way would be to do this within `startTunnel`. If process is running, just print the URL, otherwise start it. I think end users shouldn't be aware of this, esp. in their entry file!

Reply:
That makes sense! Please apply that change!